src/main.c

```diff
@@ -515,10 +515,15 @@ main(int argc, char **argv)
 
 	char *base_path = SDL_GetBasePath();
 
+    if (!base_path) {
+        base_path = SDL_strdup("./");
```

Comment:
Are you sure you need to allocate a copy of this string? The following should work:

`base_path = "./"`

This way, you don't have to SDL_free() later.